content/en/users/compute/cloud-compute/openstack/_index.md

```diff
@@ -80,7 +80,7 @@ cat /etc/grid-security/certificates/*.pem >> $(python -m requests.certs)
 ## Authentication
```

Comment:
Consider reorder this file: Put aai before OpenStack

Reply:
This page I did not touch. All pages under cloud compute that are not showing up in #411 will disappear in a future PR I am preparing.